vendor/github.com/karalabe/usb/appveyor.yml

```diff
@@ -22,8 +22,8 @@ environment:
 
 install:
   - rmdir C:\go /s /q
-  - appveyor DownloadFile https://storage.googleapis.com/golang/go1.12.5.windows-%GOARCH%.zip
```

Comment:
Please don't change things in vendored folders, those will get lost anyway on the next vendor update.

Reply:
I've merged your upstream PR into `usb`, you can update the entire vendored repo with:

```
govendor fetch github.com/karalabe/usb/^
```